testsuite/forge-cli/src/main.rs

```diff
@@ -1531,6 +1532,50 @@ fn multi_region_multi_cloud_simulation_test(config: ForgeConfig<'static>) -> For
         )
 }
 
+fn multiregion_benchmark_test(config: ForgeConfig<'static>) -> ForgeConfig<'static> {
+    ForgeConfig::default()
+        .with_initial_validator_count(NonZeroUsize::new(20).unwrap())
+        .with_network_tests(vec![&PerformanceBenchmark])
+        .with_genesis_helm_config_fn(Arc::new(|helm_values| {
+            // Have single epoch change in land blocking
+            helm_values["chain"]["epoch_duration_secs"] = 300.into();
+
+            helm_values["genesis"]["multicluster"]["enabled"] = true.into();
+        }))
+        .with_node_helm_config_fn(Arc::new(|helm_values| {
+            helm_values["multicluster"]["enabled"] = true.into();
+            // Create headless services for validators and fullnodes. 
+            // Note: chaos-mesh will not work with clusterIP services.
+            helm_values["service"]["validator"]["internal"]["type"] = "ClusterIP".into();
+            helm_values["service"]["validator"]["internal"]["headless"] = true.into();
+            helm_values["service"]["fullnode"]["internal"]["type"] = "ClusterIP".into();
+            helm_values["service"]["fullnode"]["internal"]["headless"] = true.into();
```

Comment:
nit: probably have some helper to setup the values needed for multiregion, so it can be reused in other test suites?

Reply:
I will follow-up in the next PR.